Documentation/troubleshooting.rst

```diff
@@ -132,7 +132,117 @@ e.g.:
+    Jun  2 11:14:46.043   default/tiefighter:57746                  default/deathstar-5b7489bc84-9bftc:80     to-endpoint   FORWARDED   TCP Flags: ACK, PSH
+    Jun  2 11:14:46.043   default/deathstar-5b7489bc84-9bftc:80     default/tiefighter:57746                  to-endpoint   FORWARDED   TCP Flags: ACK, PSH
+    Jun  2 11:14:46.043   default/tiefighter:57746                  default/deathstar-5b7489bc84-9bftc:80     to-endpoint   FORWARDED   TCP Flags: ACK, FIN
+    Jun  2 11:14:46.048   default/deathstar-5b7489bc84-9bftc:80     default/tiefighter:57746                  to-endpoint   FORWARDED   TCP Flags: ACK, FIN
+    Jun  2 11:14:46.048   default/tiefighter:57746                  default/deathstar-5b7489bc84-9bftc:80     to-endpoint   FORWARDED   TCP Flags: ACK
+
+You may also use ``-o json`` to obtain more detailed information about each
+flow event.
+
+In the following example the first command extracts the numeric security
+identities for all dropped flows which originated in the ``default/xwing`` pod
+in the last three minutes. The numeric security identity can then be used
+together with the Cilium CLI to obtain more information about why flow was
+dropped:
+
+.. code:: bash
+
+    $ kubectl exec -n kube-system cilium-77lk6 -- \
+        hubble observe --since 3m --type drop --from-pod default/xwing -o json | \
+        jq .destination.identity | sort -u
+    788
+
+    $ kubectl exec -n kube-system cilium-77lk6 -- \
+        cilium policy trace --src-k8s-pod default:xwing --dst-identity 788
+    ----------------------------------------------------------------
+
+    Tracing From: [k8s:class=xwing, k8s:io.cilium.k8s.policy.cluster=default, k8s:io.cilium.k8s.policy.serviceaccount=default, k8s:io.kubernetes.pod.namespace=default, k8s:org=alliance] => To: [k8s:class=deathstar, k8s:io.cilium.k8s.policy.cluster=default, k8s:io.cilium.k8s.policy.serviceaccount=default, k8s:io.kubernetes.pod.namespace=default, k8s:org=empire] Ports: [0/ANY]
+
+    Resolving ingress policy for [k8s:class=deathstar k8s:io.cilium.k8s.policy.cluster=default k8s:io.cilium.k8s.policy.serviceaccount=default k8s:io.kubernetes.pod.namespace=default k8s:org=empire]
+    * Rule {"matchLabels":{"any:class":"deathstar","any:org":"empire","k8s:io.kubernetes.pod.namespace":"default"}}: selected
+        Allows from labels {"matchLabels":{"any:org":"empire","k8s:io.kubernetes.pod.namespace":"default"}}
+          No label match for [k8s:class=xwing k8s:io.cilium.k8s.policy.cluster=default k8s:io.cilium.k8s.policy.serviceaccount=default k8s:io.kubernetes.pod.namespace=default k8s:org=alliance]
+    1/1 rules selected
+    Found no allow rule
+    Ingress verdict: denied
+
+    Final verdict: DENIED
+
+
+Please refer to the :ref:`policy troubleshooting guide<policy_tracing>` for
+more detail about how to troubleshoot policy related drops.
+
+.. note::
+    **Hubble Relay** (Beta) allows you to query multiple Hubble instances
```

Comment:
Should we document using hubble relay even if it's in beta? Because I feel that if we mention this but don't explain how, it's going to be an open question for whoever reads this doc. I would suggest to either remove this note or write doc about how to troubleshoot with hubble relay.

Reply:
:+1: When I read `In order to observe the traffic of a specific pod, you will first have to retrieve the name of the cilium instance managing it`, I immediately thought "Why can't I use Hubble relay". (But I've been coached by Robin :p)

Nit if we keep this note: `Beta` -> `beta`.

Reply:
I discussed this offline with @gandro and I'll add a hubble relay section to the doc.

Reply:
I have changed `Beta` -> `beta`, but left the note in. My thinking is that we add a link to the hubble-relay section there, but I'm also happy to remove the note completely for now.

Reply:
> My thinking is that we add a link to the hubble-relay section there

I'll do this as part of writing the hubble relay section.